src/seedu/addressbook/AddressBook.java

```diff
@@ -257,20 +264,32 @@ private static void echoUserCommand(String userCommand) {
      * @param args full program arguments passed to application main method
      */
     private static void processProgramArgs(String[] args) {
-        if (args.length >= 2) {
+        if (isProgramValid(args)) {
```

Comment:
The name here is misleading. The function will return true when the number of args is invalid.